plugins/by-name/oil-git-status/default.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "oil-git-status";
+  packPathName = "oil-git-status.nvim";
+  package = "oil-git-status-nvim";
+
+  description = ''
+    Add Git Status to oil.nvim directory listings.
+    Git status is added to the listing asynchronously after creating the oil directory
```

Is this intended to render as a newline or paragraph break?

If so you need either:

```suggestion
    Add Git Status to oil.nvim directory listings. \
    Git status is added to the listing asynchronously after creating the oil directory
```

for a newline, or:

```suggestion
    Add Git Status to oil.nvim directory listings.

    Git status is added to the listing asynchronously after creating the oil directory
```

for a paragraph break.